cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/healthcheck/HealthCheckDialog.java

```diff
@@ -78,7 +78,7 @@ public void execute() {
       this.report = performHealthCheckReport();
 
       if (generatorConfProj != null && generatorConfProj.getLocationURI() != null) {
-        CobiGenFactory.create(generatorConfProj.getLocationURI());
+        CobiGenFactory.create(generatorConfProj.getLocationURI(), true);
```

Comment:
I'm not sure about this. The healthCheck should react to monolithic templates and suggest an upgrade I think.

Reply:
This mean, inside the healthCheck I have to create a new executeWithMonolithicTemplates Method to run the old tests correctly, is that correct ?